cli/src/commands/branch.rs

```diff
@@ -107,6 +111,56 @@ pub struct BranchSetArgs {
     pub names: Vec<String>,
 }
 
+/// Start tracking given remote branches
+///
+/// A tracking remote branch will be imported as a local branch of the same
+/// name. It will be merged with the existing local branch on future pulls.
```

Comment:
"merge with" is probably too technical here. Maybe something like "Changes to it will propagate to the existing local branch on future pulls"?

Reply:
> Changes to it will propagate to the existing local branch on future pulls
Sounds good. Fixed the others, too, thanks.